### PR TITLE
Feature/11 setup nextjs project

### PR DESCRIPTION
**Pull request by modous**
https://github.com/users/modous/projects/6/views/1?visibleFields=%5B%22Title%22%2C%22Assignees%22%2C%22Status%22%2C76985701%2C76985703%2C76985702%2C76985704%2C%22Labels%22%5D&pane=issue&itemId=52761972

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import Image from "next/image";
+import styles from "./page.module.css";
+
+export default function Home() {
+  return (
+    <main className={styles.main}>
```
Comment (jaerpollux) [inline src/app/page.tsx]: You can remove the boilerplate code.

File: src/app/page.module.css
Comment (jaerpollux) [inline src/app/page.module.css:1]: You can remove the boilerplate code.

File: public/vercel.svg
Comment (jaerpollux) [inline public/vercel.svg:1]: You can remove the boilerplate images.

File: public/next.svg
Comment (jaerpollux) [inline public/next.svg:1]: You can remove the boilerplate images.

**Review by jaerpollux — CHANGES_REQUESTED** [review]

**Review by peerjollux — APPROVED** [review]

**Pull request merged**